apps/server/src/modules/h5p-editor/controller/dto/h5p-editor.params.ts

```diff
@@ -14,6 +16,18 @@ export class GetH5PContentParams {
 	contentId!: string;
 }
 
+export class GetH5PEditorParams {
+	@ApiPropertyOptional()
+	@IsMongoId()
+	contentId?: string;
```

Comment:
```suggestion
	contentId?: string;
```

If you check @IsMongoId(), then it is not optional